src/kili/adapters/kili_api_gateway/asset/formatters.py

```diff
@@ -3,18 +3,27 @@
 import json
 from typing import Dict
 
+import requests
+
 from kili.adapters.http_client import HttpClient
 from kili.core.helpers import is_url
 from kili.domain.types import ListOrTuple
+from kili.log.logging import logger
 
 
 def load_json_from_link(link: str, http_client: HttpClient) -> Dict:
     """Load json from link."""
     if link == "" or not is_url(link):
         return {}
 
+    response = http_client.get(link, timeout=30)
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        logger.exception("An error occurred while processing the response: %s", err)
+        raise RuntimeError(f"An error occurred while processing the response: {err}") from err
     try:
-        return http_client.get(link, timeout=30).json()
+        return response.json()
     except json.JSONDecodeError:
         return {}
```

Comment:
we should log here maybe, if the json can't be parsed (not sure why/when it happens...?)